docs/bloodmoon-cult/bloodpriest.mdx

```diff
@@ -30,6 +27,6 @@ The Blood Priest is a member of the Bloodmoon Cult with the ability to mark play
 
 ## History
 
-<Timeline lineColor="white">
-  <Event interval="2019-01-01">Introduced in ext-300</Event>
-</Timeline>
+<timeline>
```

Comment:
Convention in their docs is to lowercase, plus it avoids clashes with our wrapper component.